blob/azureblob/azureblob.go

```diff
@@ -482,11 +487,12 @@ func openBucket(ctx context.Context, pipeline pipeline.Pipeline, accountName Acc
 	}
 	serviceURL := azblob.NewServiceURL(*blobURL, pipeline)
 	return &bucket{
-		name:         containerName,
-		pageMarkers:  map[string]azblob.Marker{},
-		serviceURL:   &serviceURL,
-		containerURL: serviceURL.NewContainerURL(containerName),
-		opts:         opts,
+		name:                 containerName,
+		pageMarkers:          map[string]azblob.Marker{},
+		serviceURL:           &serviceURL,
+		containerURL:         serviceURL.NewContainerURL(containerName),
+		opts:                 opts,
+		credentialExpiration: time.Time{},
```

Comment:
Nit: this shouldn't be necessary, time.Time{} is the default value for time.Time.